src/layers/others.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    LayerScale(scale) 
+
+Implements LayerScale.
+([reference](https://arxiv.org/abs/2103.17239))
+
+# Arguments
+- `scale`: Scaling factor, a learnable diagonal matrix which is multiplied to the input.
+"""
+struct LayerScale
+  scale
+end
+
+"""
+    LayerScale(λ, planes::Int)
+
+Implements LayerScale.
+([reference](https://arxiv.org/abs/2103.17239))
+
+# Arguments
+- `planes`: Size of channel dimension in the input.
+- `λ`: initialisation value for the learnable diagonal matrix.
+"""
+LayerScale(planes::Int, λ) = λ > 0 ? LayerScale(Flux.ones32(planes) * λ) : identity
+
+@functor LayerScale
+(m::LayerScale)(x::AbstractArray{T}) where T = convert.(T, m.scale) .* x
+
+"""
+    DropPath(p)
+
+Implements Stochastic Depth.
+([reference](https://arxiv.org/abs/1603.09382))
+
+# Arguments
+- `p`: rate of Stochastic Depth.
+"""
+DropPath(p) = p ≥ 0 ? Dropout(p; dims = 4) : identity
```

Comment:
Not a constructor, so should be lowercase.

Reply:
This was more keeping the semantics of layers in mind: I'll make the change if it's a strict rule but given the usage I thought it would be more appropriate this way

Reply:
> doesn't really meet the threshold a layer by Flux standards

Yeah this is why I kept it in Metalhead itself (same as for `ChannelLayerNorm`)  - I don't mind making it the direct version either, I just thought this helps Julian first-timers understand a little better how this works

Reply:
Yeah but `ChannelLayerNorm` has a forward pass that isn't just a chain of existing layers (in this case it isn't even a `Chain` but convenience for a single keyword).

Reply:
Yep, done now